public/diagnostic.js

```diff
@@ -23,3 +23,34 @@
 //  -  Order 1 : 2 hats ($5 each)
 //  -  Order 2 : 1 desk lamp ($20 each)
 //  -  Order 3 : 3 hand towels ($8 each)
+
+const Order = Ember.Object.extend({
+  unitPrice: NaN,
+  quantity: NaN,
+  name: '',
+  orderPrice: Ember.computed('unitPrice', 'quantity', function() {
+    return unitPrice * quantity;
+  }),
+});
```

Comment:
in Ember we are using getter and setters, so instead of `unitPrice * quantity` we would return `this.get('price') * this.get('quantity')`